Sources/Bauletto/BaulettoSettings.swift

```diff
@@ -27,20 +27,27 @@ public struct BaulettoSettings {
     /// Type of the haptic feedback fired when the banner is shown. None by default.
     public var hapticStyle: FeedbackGenerator.HapticStyle = .none
     
-    public init(icon: UIImage?, title: String?, backgroundStyle: UIBlurEffect.Style = .regular, dismissMode: BaulettoDismissMode = .automatic, hapticStyle: FeedbackGenerator.HapticStyle = .none) {
+    /// The optional action that needs to be performed when the view is tapped.
+    public var action: (() -> Void)?
+    
+    public init(icon: UIImage?, title: String?, backgroundStyle: UIBlurEffect.Style = .regular, dismissMode: BaulettoDismissMode = .automatic, hapticStyle: FeedbackGenerator.HapticStyle = .none,
+                action: (() -> Void)? = nil) {
```

Comment:
```suggestion
    public init(icon: UIImage?, title: String?, backgroundStyle: UIBlurEffect.Style = .regular, dismissMode: BaulettoDismissMode = .automatic, hapticStyle: FeedbackGenerator.HapticStyle = .none, action: (() -> Void)? = nil) {
```

Can you please leave it on one live? Thanks!